src/test/fuzz/fuzz.cpp

```diff
@@ -4,10 +4,14 @@
 
 #include <test/fuzz/fuzz.h>
 
+#include <netaddress.h>
+#include <netbase.h>
 #include <test/util/setup_common.h>
 #include <util/check.h>
+#include <util/sock.h>
 
 #include <cstdint>
+#include <memory>
```

Comment:
A call to `std::terminate()` was added, which is defined in `<exception>`.
```suggestion
#include <exception>
#include <memory>
```

Reply:
Oh, thanks! Now fixed.